testdata/long_gen

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/python2.7
+
+# generate superlong testcase
+
+from random import randint
+a = []
+i = 0
+while i < 2147483:
+    a.append(randint(-2147483648,2147483647))
```

Comment:
This script isn't too helpful from a C.I perspective seeing as if something like this was committed to the tree it (and the associated test) probably wouldn't ever be run again given that the test has to be enabled manually. This makes it about as useful as if it wasn't added at all.
My suggestion for something like this is to have the script run when CMake is run, and for it to pass a seed into the random number generator so that the results are the same across multiple runs (otherwise test failures could be sporadic). 
Also, if this is in the testdata folder I would document it a little more, including for example a brief description of the output. In the context of the entire project to say that it generates a 'testcase' doesn't mean much.